DATA/UNIVERSE/SYSTEMS/Bw10/Bw10.ini

```diff
@@ -1107,24 +1107,28 @@ encounter = tradelane_trade_transport, 19, 0.11
 faction = co_ti_grp, 0.35 ; Planetform, Inc.
 
 [Object]
-nickname = Bw10_Trade_Lane_Ring_Construct_A_1
+nickname = Bw10_Trade_Lane_Ring_19
 ids_name = 261692
+;res str $jumpgate_tau23_name

```

Comment:
This is wrong here. The ids_name is actually just Trade Lane Ring. As the vanilla IDS. The game uses the tradelane_space_name only for properly displaying the direction.

Reply:
Actually, we are both wrong. In vanilla all of the ids_name entries used are supposed to be "\ \."
The 4th tradelane in this system references the ids_name for the Omicron Alpha Jump Hole
I don't understand the significance of there being many ids_name infocard entries with the same content: \ \.